src/index.ts

```diff
@@ -1,44 +1,17 @@
-// tslint:disable:no-console no-var-requires no-any
+// tslint:disable:no-console no-any
 import chalk from 'chalk';
 import * as fs from 'fs';
 import * as glob from 'glob';
 import {Plugin as PostCssPlugin} from 'postcss';
 import {Tapable} from 'tapable';
 import {promisify} from 'util';
 import {Compiler} from 'webpack';
-
-const cssModuleCore = require('css-modules-loader-core');
-const DtsCreator = require('typed-css-modules');
-const FileSystemLoader = require('typed-css-modules/lib/fileSystemLoader').default;
+import cssModuleCore = require('css-modules-loader-core');
```

Comment:
can you use `import * as cssModuleCore from 'css-modules-loader-core'`?

Reply:
i think there was some issue with that. but let me try again.

Reply:
it is possible. my bad. i had tried `esModuleInterop` but this would have changed the emit.
unfortunately `DtsCreator` has to be imported with require, as it is defined in CommonJS style `export = ...`.